tracee-ebpf/main.go

```diff
@@ -955,6 +958,20 @@ func prepareEventsToTrace(eventFilter *tracee.StringFilter, setFilter *tracee.St
 	return res, nil
 }
 
+var necessaryCapabilities = [...]capability.Cap{capability.CAP_SYS_ADMIN, capability.CAP_IPC_LOCK, capability.CAP_SYS_PTRACE, capability.CAP_BPF, capability.CAP_PERFMON}
+
+func dropUnnecessaryCapabilities(caps capability.Capabilities) error {
```

Comment:
`dropUnneededCapabilities()` ?

Reply:
Note: you should stash your changes to the same logic in the same PR. One commit should contain either one feature or one fix. If you have multiple commits doing the same thing you should stash them together before the PR. 

Reason: this way you remove the burden of making the reviewer to review something about to be changed over and over in the same PR.